afids_utils/afids.py

```diff
@@ -0,0 +1,59 @@
+"""Anatomical fiducial classes"""
+from __future__ import annotations
+
+import attr
```

Comment:
[I think `attrs` is the preferred namespace](https://www.attrs.org/en/stable/names.html#tl-dr)